data/server_locations.json

```diff
@@ -52665,6 +52665,52 @@
                 "id": 3352,
                 "last_updated": "2024-05-22"
             }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    4.637334800000001,
+                    52.3810724
+                ]
+            },
+            "properties": {
+                "name": "Grote Kerk ",
+                "area": "Netherlands",
+                "address": "Grote Markt 22, Haarlem ",
+                "status": "unvisited",
+                "external_url": "http://209.221.138.252/Details.aspx?location=430006",
+                "internal_url": "null",
+                "latitude": "52.3810724",
+                "longitude": "4.637334800000001",
+                "machine_status": "available",
+                "id": 7187,
+                "last_updated": "2024-05-10"
+            }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    19.699024,
+                    48.669026
+                ]
+            },
+            "properties": {
+                "name": "Please rename this country",
```

Comment:
@jannisborn this seems to be a wrong new machine from the website, are we moving it to the skip.json file with the ignored machines?